### DISCUSSION TITLE
GitHub deployments appear after months of git.deploymentEnabled = false

### DISCUSSION THREAD

**Discussion by rnystrom** [vercel/community#3009] [Bug Report]
### Page to Investigate

n/a

### Steps to Reproduce

We have a monorepo with 2 vercel projects in it. Each have a `vercel.json` with:

```json
{
  "github": {
    "silent": true
  },
  "git": {
    "deploymentEnabled": false
  }
}
```

We've had this setup for almost a 6 months and using the CLI to deploy with our custom setup. However starting today we're getting Vercel deployments and comments on our PRs again. Did something change or do we need to update a config?

<img width="941" alt="image" src="https://github.com/vercel/community/assets/739696/80f54da6-010a-4a14-bf50-6f607fe0a373">

<img width="936" alt="image" src="https://github.com/vercel/community/assets/739696/3c4974fb-31d5-4859-9203-b1d1bb26757d">

**Comment by yashsway** [comment]
Getting the same issue as of this morning in my org.

**Comment by RyanCCollins** [comment]
Same!

**Comment by ofhouse** [comment]
Thanks for letting us know, we're investigating a potential regression on this!

**Comment by JTaylor0196** [comment]
Hey @rnystrom, @yashsway, @RyanCCollins,

A fix should be in place for this now. If you're still running into issues, let us know!

**Reply by rnystrom** [reply]
Work on our end now, thank you!

**Reply by yashsway** [reply]
working on my end as well! thank you!
